### ISSUE TITLE
1.18.1 ver 1.12.10 crash

### ISSUE THREAD

**Issue by KentaJin** [Chocohead/OptiFabric#536]
Everytime game starts up, it crash with no report and just Code:0

Installed:
• fabric-api-0.44.0+1.18
• OptiFine_1.18.1_HD_U_H4
• fabric loader 0.12.12 - 1.18.1
• Origin Enhanced

**Comment by ghost** [comment]
Just ran a comparison against what im running, the only difference is my fabric loader, which is 0.12.11, and Optifine as I tried the h5 Pre2 beta. Are you running any other mods at all? can you also provide a copy of your logs?

**Comment by KentaJin** [comment]
> 

I use another mod name Origins Enhanced now that I remember, it is for Origin realms server
https://www.curseforge.com/minecraft/mc-mods/origins-enhanced
if you talk about crash log the game generated non
Two mods work together normally for months now so i dont think it cause problem now

**Comment by ghost** [comment]
Yeah theres another report saying origins isn't working with Optifabric, so i'd suggest trying without, and if it works thats the issue. Im running all the same stuff, as well as Shulker Tooltip, Litematica, Xaero's world map, AuthMe, WTHIT, except the only difference is im running h5 pre2, and Optifabric loads fine.

**Comment by IllagerCaptain** [comment]
Origins has nothing to do with Origins Enhanced @MrFawkes1337.

**Comment by KentaJin** [comment]
> Yeah theres another report saying origins isn't working with Optifabric, so i'd suggest trying without, and if it works thats the issue. Im running all the same stuff, as well as Shulker Tooltip, Litematica, Xaero's world map, AuthMe, WTHIT, except the only difference is im running h5 pre2, and Optifabric loads fine.

then pretty sure the crash cause by the incompatible between optifabric and optifine atm

**Comment by KentaJin** [comment]
Okay so i have some test and seem like it was cause by incompatible between optifabric and origin enhanced

**Comment by ghost** [comment]
> Origins has nothing to do with Origins Enhanced @MrFawkes1337.

Seems I was right however, as origins enhanced was his issue. I’m not too clued on all the mods publicly available, so just took a guess that was right.

**Comment by IllagerCaptain** [comment]
> Okay so i have some test and seem like it was cause by incompatible between optifabric and origin enhanced

I can also confirm this.
